common/src/test/java/uk/gov/justice/services/common/converter/ObjectToJsonObjectConverterTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.justice.services.common.converter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.datatype.jsr353.JSR353Module;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.isA;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.doThrow;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ObjectToJsonObjectConverterTest {
+
+    private static final UUID ID = UUID.randomUUID();
+    private static final String NAME = "Pojo";
+    private static final List<String> ATTRIBUTES = Arrays.asList("Attribute 1", "Attribute 2");
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Mock
+    private ObjectMapper mapper;
+
+    @Test
+    public void shouldConvertPojoToJsonObject() throws Exception {
```

Comment:
Again if not testing the returned exception explicitly I think the expected exception.class in the annotation is clearer. Perhaps we should have a discussion about @Rule so we have a consistent approach?